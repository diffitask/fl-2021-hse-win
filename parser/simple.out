Alphabet -- 2 symbol(s):
             (a, b)
Start state:
             0
Terminals -- 1 state(s):
             1
All states:
           (number: 0, type: start)
           (number: 1, type: terminal)
Edges:
            0 --> 1, symbols: (a, b)
