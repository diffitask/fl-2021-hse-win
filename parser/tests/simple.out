Alphabet -- 2 symbol(s):
             (a, b)
Start state:
             1
Terminals -- 1 state(s):
             2
All states:
           (number: 1, type: start)
           (number: 2, type: terminal)
           (number: 3, type: average)
           (number: 4, type: runoff)
Edges:
            1 --> 2, symbols: (a)
            1 --> 3, symbols: (b)
            2 --> 2, symbols: (a, b)
            3 --> 4, symbols: (a, b)
            4 --> 4, symbols: (a, b)
